enterprise/startup-script/nginxconfiguressl.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+export STORAGE_ACCESS_ACCOUNT=$1
+export STORAGE_ACCESS_KEY=$2
+export FQDN=$3
+export certprint=$4
+export cacerts=$5
+
+sudo docker run -d --name scalityconnect -p 8000:8000 \
+-p 8005:8005 -e FQDN=$FQDN \
+-e AZURE_STORAGE_ACCESS_KEY=$STORAGE_ACCESS_KEY \
+-e AZURE_STORAGE_ACCOUNT=$STORAGE_ACCESS_ACCOUNT \
+--log-opt max-size=5g --log-opt max-file=10 \
+--restart always scality/connect
+
+# exit on any error
+set -e
+
+until apt-get -y update && apt-get -y install nginx
+do
+    echo "Trying again"
+    sleep 2
+done
+
+sslcertfilename=$certprint'.crt'
+sslkeyfilename=$certprint'.prv'
+
+echo "Copying SSL files"
+echo "cert file" $sslcertfilename
+fullpath=/var/lib/waagent/$sslcertfilename
+if [ -f $fullpath ]
+then
+    cp $fullpath /etc/ssl/certs/
+    echo "Getting CA SSL files"
+    wget $cacerts -O 'CASignedCert'$certprint
+    chmod 777 /etc/ssl/certs/$sslcertfilename
+    cat 'CASignedCert'$certprint >> /etc/ssl/certs/$sslcertfilename
+    rm -f 'CASignedCert'$certprint
+else
+    echo "Cert missing: " $fullpath
+    exit 1
+fi
+
+
+echo "private key file" $sslkeyfilename
+fullpath=/var/lib/waagent/$sslkeyfilename
+if [ -f $fullpath ]
+then
+    cp $fullpath /etc/ssl/private/
+else
+    echo "Private Key missing: " $fullpath
+    exit 1
+fi
+
+echo "configuring certfile"
+mv /var/scality/ssl.conf /etc/nginx/snippets/ssl.conf
+echo "configuring certfile"
+sed -i 's/${SSL_CERT_FILE_NAME}/'$sslcertfilename'/' /etc/nginx/snippets/ssl.conf
+echo "keyfile"
+sed -i 's/${SSL_KEY_FILE_NAME}/'$sslkeyfilename'/' /etc/nginx/snippets/ssl.conf
+
+echo "configuring params.conf"
+openssl dhparam -out /etc/ssl/certs/dhparam.pem 2048
+mv /var/scality/ssl-params.conf /etc/nginx/snippets/ssl-params.conf
+
+echo "updating main conf file"
+mv /etc/nginx/nginx.conf /etc/nginx/nginx.conf.default
+mv /var/scality/nginx.conf.template /etc/nginx/nginx.conf
+
+sed -i -e 's/\${UPSTREAM_SERVER}/localhost:8000/' /etc/nginx/nginx.conf
+sed -i -e 's/\${LISTEN_PORT}/80/' /etc/nginx/nginx.conf
+sed -i -e 's|\${FQDN}|'"$FQDN"'|' /etc/nginx/nginx.conf
+
+echo "updating sites-enabled/default file"
+mv /etc/nginx/sites-enabled/default /etc/nginx/sites-enabled/default.default
+mv /var/scality/default.template.conf /etc/nginx/sites-enabled/default
+sed -i -e 's|\${FQDN}|'$FQDN'|' /etc/nginx/sites-enabled/default
+
+systemctl restart nginx
```

Comment:
what happens if the system is restarted or nginx goes down? docker is taking care of restarting scality connect for us.